paddlenlp/transformers/electra/modeling.py

```diff
@@ -585,6 +592,18 @@ def forward(self,
                 When the data type is float, the `masked` tokens have `-INF` values and the others have `0` values.
                 It is a tensor with shape broadcasted to `[batch_size, num_attention_heads, sequence_length, sequence_length]`.
                 Defaults to `None`, which means nothing needed to be prevented attention to.
+            past_key_values (tuple(paddle.nn.MultiHeadAttention.Cache), optional):
```

Comment:
Addressed.
One minor thing that is different from HF is that here we have 2 tensors in `past_key_values` instead of 4 tensors (key, value, cross attention key, cross attention values). Since electra is encoder-only, there is no cross attention key and values. This way we can directly use `Cache` object and don't need to bother with the `StaticCache` object